src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -131,16 +131,24 @@
       + "`kafka`\n"
       + "    Must be a trio representing the Kafka coordinates, defaults to ``" + StringUtils.join(DEFAULT_KAFKA_PK_NAMES, ",") + "`` if empty.\n"
       + "`record_key`\n"
-      + "    If empty, all fields from the key struct will be used, otherwise used to whitelist the desired fields - for primitive key only a single field name must be configured.\n"
+      + "    If empty, all fields from the key struct will be used, otherwise used to extract the desired fields - for primitive key only a single field name must be configured.\n"
       + "`record_value`\n"
-      + "    If empty, all fields from the value struct will be used, otherwise used to whitelist the desired fields.\n";
+      + "    If empty, all fields from the value struct will be used, otherwise used to extract the desired fields.";
   private static final String PK_FIELDS_DISPLAY = "Primary Key Fields";
 
+  public static final String FIELDS_WHITELIST = "fields.whitelist";
```

Comment:
Hmm, I'm realizing this is a bit inconsistent with the PK version since this uses `whitelist` whereas the PK one just says `pk.fields`. Should we try to unify them on `whitelist` or does that just add noise?